flask_resty/testing.py

```diff
@@ -30,30 +30,37 @@ def get_meta(response):
 def assert_value(actual, expected):
     if isinstance(expected, Mapping):
         assert isinstance(actual, Mapping)
-        for k, v in expected.items():
-            assert_value(actual.get(k, None), v)
+        # Unlike all the others, this checks that the actual items are a
+        # superset of the expected items, rather than that they match.
+        for key, value in expected.items():
+            if value is not UNDEFINED:
```

Comment:
I don't really think this is the best approach, UNDEFINED shouldn't be a special case handled here, but rather we should have a more stricter ANY that checks for nullness. This code is not aware of any, and it shouldn't be aware of UNDEFINED either